runtime/lib/ttnn/operations/eltwise/unary.cpp

```diff
@@ -1,6 +1,8 @@
 // SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
 //
 // SPDX-License-Identifier: Apache-2.0
+#define TRACY_ENABLE 1
```

Comment:
Should we wrap this inside a check of: `#if defined(TT_RUNTIME_ENABLE_PERF_TRACE)`

Reply:
yeah, actually this should be a higher level compile define from root of project now that I think about it. Let me update

Reply:
updated. I placed it in root of the project as recommend in the Tracy Documentation on how to inherit Tracy into other projects.